tests/dags/providers/provider_api_scripts/test_inaturalist.py

```diff
@@ -167,6 +167,8 @@ def test_consolidate_load_statistics(all_results, expected):
     ],
 )
 def test_get_batches(batch_length, max_id, expected):
-    with mock.patch.object(PostgresHook, "get_records", return_value=max_id) as pg_mock:
-        actual = INAT.get_batches(batch_length, pg_mock)
-        assert actual == expected
+    task = "doesn't matter, but we have to pass something"
```

Comment:
In this case, you can just use the following, as it gets the point across 🙂 
```suggestion
    task = mock.Mock()
```